### PR TITLE
Add compatibility for kubernetes.io/basic-auth secret types

### PR DESCRIPTION
**Pull request by Michkov**
Allow to pass kubernetes.io/basic-auth with username and password keys as workspace to git-clone task.

### CODE REVIEW

File: task/git-clone/0.1/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ The git-clone Task will clone a repo from the provided url into the output Works
 |verbose|Log the commands that are executed during `git-clone`'s operation.|true|false|
 |gitInitImage|The image providing the git-init binary that this Task runs.|registry.redhat.io/openshift-pipelines/pipelines-git-init-rhel8:v1.8.2-8@sha256:a538c423e7a11aae6ae582a411fdb090936458075f99af4ce5add038bb6983e8|false|
 |userHome|Absolute path to the user's home directory. Set this explicitly if you are running the image as a non-root user or have overridden the gitInitImage param with an image containing custom user configuration. |/tekton/home|false|
-|enableSymlinkCheck|Check symlinks in the repo. If they're pointing outside of the repo, the build will fail.|true|false|
+|enableSymlinkCheck|Check symlinks in the repo. If they're pointing outside of the repo, the build will fail. |true|false|
```
Comment (mkosiarc) [inline task/git-clone/0.1/README.md:23]: why this change?

Comment (Michkov) [reply task/git-clone/0.1/README.md:23]: I run script to update documentation, so this was updated. Let me change.

**Review by mkosiarc — APPROVED** [review]

**Comment by Michkov** [comment]
known enterprise contract issue.
/retest
